microprot/snakemake/Snakefile

```diff
@@ -3,192 +3,276 @@
 # "qsub -k eo -m n -l nodes=1:ppn={cluster.n} -l mem={cluster.mem}gb \
 # -l walltime={cluster.time}" --directory "$@"
 
-import os
-import skbio
+# snakemake -s Snakefile_minimal -n
+
 import yaml
 import re
-from microprot.scripts import split_sequence, mask_sequence, calculate_Nf
+import sys
+from glob import glob
+from os import makedirs
+from os.path import basename, splitext, exists
+from shutil import copyfile
+from skbio import io
+
+sys.path.append('/projects/microprot')
```

Comment:
OK, input/output split implemented.
Need a few more minutes for `replace`. Is not as trivial as it seems.;)